### ISSUE TITLE
How to download the dataset via wget?

### ISSUE THREAD

**Issue by davide-coccomini** [yinanhe/ForgeryNet#19]
Good morning.
I am trying to download the training and validation set. I got access to Google Drive but I need to be able to download it via wget. I can't because authentication is required and then via wget I am redirected to the login page.
I have also tried using the 105 site but I cannot register because they do not accept Italian phone numbers.

Do you have any alternatives to allow me to download via wget?

**Comment by yinanhe** [comment]
Maybe you can use [gdown](https://github.com/wkentaro/gdown)

**Comment by davide-coccomini** [comment]
I tried with gdown and I can't do that since it only works with public files. I tried gshell but even if it allows you to download private files, you still can not download from shared folders.

**Comment by davide-coccomini** [comment]
I solved it by buying space on Google Drive (given the large size of the considered files) and using the 'create a copy' option to move the files to my personal Drive. After doing this I was able to download the files via [gshell](https://github.com/wkentaro/gshell).

Obviously, it would be better if there was a solution that didn't involve spending money, however little, but for those who are experiencing the same problem, this is a possible solution.
